taskfile.yaml

```diff
@@ -121,14 +116,12 @@ tasks:
       - "clp-py-utils"
       - "init"
       - "job-orchestration"
-      - "log-viewer-webui"
-      - "nodejs-14"
       - "package-venv"
+      - "webui"
       - task: "utils:checksum:validate"
```

Comment:
Alphabetize?

Reply:
1. since the order of `deps` tasks don't really matter, would it be fine to leave it given the last task is multiline?
2. if not, shall we add `task:` to the "webui" string / all strings to make the formatting consistent?